.github/workflows/deps_lint.yml

```
@@ -87,6 +87,31 @@ jobs:
         .ci/openapi_diff.sh
         .ci/index_openapi_diff.sh
 
+  docker-image:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Build the Docker image
+      run: docker-compose build
+
+    - name: Test server
+      run: |
+        docker-compose up optimade &
+        .github/workflows/wait_for_it.sh localhost:3213 -t 120
+        sleep 15
+        curl http://localhost:3213/optimade/info > info.json
+        grep -F "http://example.com" info.json || exit 1
```

Actually, is there any reason we couldn't use the validator here? I'm assuming the container name used here is deterministic...

```suggestion
        docker exec optimade-python-tools_optimade_1 optimade_validator http://localhost:5000/optimade/v0
```